app/controllers/branches_controller.rb

```diff
@@ -39,7 +39,7 @@ def create
 
   def check_password
     unless Admin.authenticate(params[:password])
-      flash[:error] = "Incorrect password for deletion"
+      flash[:error] = 'Incorrect password for deletion'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.